html/global_attributes.json

```diff
@@ -37,6 +37,39 @@
           }
         }
       },
+      "anchor": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/anchor",
```

Comment:
IMO, it would be good to specify the flag information that is needed to use the feature as well:

```
"flags": [
  {
    "type": "preference",
    "name": "Experimental Web Platform features",
    "value_to_set": "Enabled"
  }
]
```

Same with the `HTMLElement.anchorElement` property?